vagrant/roles/logs.collect/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Create log directory structure
+  file:
+    path: "{{ config.statedir }}/{{ inventory_hostname }}"
+    state: directory
+  delegate_to: localhost
+
+- name: Copy /var/log directory
+  synchronize:
+    src: /var/log
+    dest: "{{ config.statedir }}/{{ inventory_hostname }}"
+    mode: pull
+
+- name: Process backend specific tasks
+  include_role:
+    name: 'sit.{{ config.be.name }}'
+    tasks_from: logs/main.yml
+
+- name: Copy journal
+  block:
+    - name: Extract journal since last boot
+      shell: journalctl -xb -n all > journal
```

Comment:
Why not just extract it to /var/log/journal and move the task before copying /var/log ?

Reply:
Done.